src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -50,7 +51,21 @@ public void execute_allFieldsSpecifiedUnfilteredList_success() {
     }
 
     @Test
-    public void execute_someFieldsSpecifiedUnfilteredList_success() {
+    public void execute_allFieldsSpecifiedUnfilteredListByName_success() {
```

Comment:
Can there be equivalent test cases for failures of by name?

Reply:
added!